gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -223,6 +229,25 @@ public void execute() throws MojoExecutionException {
     }
   }
 
+  private void iterateBySimulations(Toolchain toolchain, List<String> jvmArgs, List<String> testClasspath, Collection<String> simulations) throws Exception {
+    Exception[] exc = new Exception[1];
+    for (String simulation : simulations) {
+      try {
+        executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+      } catch (Exception e) {
+        if (continueAfterFail) {
```

Comment:
You didn't take my previous comment about fast exiting into account. If the first failing simulation is the last one, there's no reason to log the WARN message below.